packages/coinjoin/src/client/CoinjoinRound.ts

```diff
@@ -114,41 +120,18 @@ export class CoinjoinRound extends EventEmitter implements SerializedCoinjoinRou
         accounts: Account[],
         statusRounds: Round[],
         coinjoinRounds: CoinjoinRound[],
+        prison: CoinjoinPrison,
         options: CoinjoinRoundOptions,
     ) {
-        // TODO: this code will be removed with "selectRound" implementation
-        if (coinjoinRounds.find(r => r.id === 'fakeroundid') || accounts.length === 0) {
-            return;
-        }
-        const firstRound = statusRounds.find(r => r.phase === RoundPhase.InputRegistration);
-        if (!firstRound) return;
-        const round = new CoinjoinRound(firstRound, options);
-
-        round.startFakeRoundLifecycle(firstRound, accounts[0]);
-        return Promise.resolve(round);
-    }
-
-    // temporary code to run Round lifecycle without actual coinjoining
-    startFakeRoundLifecycle(round: Round, account: Account) {
-        console.warn('Create fake round');
-        this.id = 'fakeroundid';
-        this.inputs = account.utxos.map(u => ({
-            outpoint: u.outpoint,
-            path: u.path,
-            accountKey: account.accountKey,
-        }));
-        let currentPhase = 0;
-        const timeoutFn = async () => {
-            await this.onPhaseChange({ ...round, phase: currentPhase });
-            await this.process([]);
-
-            currentPhase++;
-            if (this.failed.length === 0 && currentPhase <= RoundPhase.Ended) {
-                setTimeout(() => timeoutFn(), 10000);
-            }
-        };
-
-        timeoutFn();
+        return selectRound(
+            (...args: ConstructorParameters<typeof CoinjoinRound>) => new CoinjoinRound(...args),
+            (...args: ConstructorParameters<typeof Alice>) => new Alice(...args),
+            accounts,
+            statusRounds,
+            coinjoinRounds,
+            prison,
+            options,
+        );
```

Comment:
nit:
would be a bit more readable if selectRound accepts params as one object so the parameters are named:
```suggestion
        return selectRound({
            roundGenerator: (...args: ConstructorParameters<typeof CoinjoinRound>) =>
                new CoinjoinRound(...args),
            aliceGenerator: (...args: ConstructorParameters<typeof Alice>) => new Alice(...args),
            accounts,
            statusRounds,
            currentRounds: coinjoinRounds,
            prison,
            options,
        });
```

Reply:
followup

Reply:
followup here https://github.com/trezor/trezor-suite/issues/6698